.prettierrc

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
please add a prettier runner in grunt. it's not all IDE that will parse and correct the files and our stack should be IDE independent.